app/Commands/LaunchCommand.php

```diff
@@ -43,28 +43,35 @@ public function handle()
                 } else return Command::SUCCESS;
             }
 
-            // 1. create a fly app, including asking the app name and setting a region
+            // 1. Create a fly app, including asking the app name and setting a region
             $appNameInput = $this->ask("Choose an app name (leave blank to generate one)"); //not putting --generate-name as the default answer to prevent it being displayed in the prompt
             $appName = '';
 
             $this->task("Create app on Fly.io", function() use($appNameInput, &$appName) {
                 $appName = $this->createApp($appNameInput);
                 return true;
             });
-
+            
             $nodeVersion = "";
             $phpVersion = "";
 
-            // 2. detect Node and PHP versions
+            // 2. Detect Node and PHP versions
             $this->task("Detect Node and PHP versions", function() use(&$nodeVersion, &$phpVersion) {
                 $nodeVersion = $this->detectNodeVersion();
                 $phpVersion = $this->detectPhpVersion();
                 return true;
             });
 
+            // 3. Ask user for processes to add 
+            $processes = [];
+            $this->task("Set additional process groups", function() use(&$processes) {
+                $processes = $this->setProcesses();
+                return true;
+            });
+
             // 3. Generate fly.toml file
```

Comment:
change this number to 4 and increment all the others too

Reply:
this makes sense! will move it

Reply:
@Johannes-Werbrouck made some changes thanks to your comments, let me know if any additional revisions is needed!